doc/modules/ROOT/pages/quickstart.adoc

```diff
@@ -0,0 +1,178 @@
+= Quickstart
+:description: This tutorial introduces you to Hazelcast's .NET client. At the end of this tutorial, you'll know how to install the client and connect it a Hazelcast cluster, work with an in-memory storage such as a map, and perform queries using SQL.
+
+Hazelcast .NET client brings the full power of the Hazelcast high-performance, in-memory computing platform to the Microsoft .NET ecosystem.
+The client allows you to elastically scale your .NET caches at high read speeds, to access all of Hazelcast data structures
+such as distributed maps, queues, topics and more. All, with enterprise level security through SSL and mutual authentication.
```

Comment:
I think it contradicts with the content. Although it mentions data structures, the guide is only about connecting to the cluster.